src/main/java/co/rsk/federate/BtcToRskClient.java

```diff
@@ -206,35 +215,44 @@ public void updateBridge() {
 
         // Call receiveHeaders
         try {
-            int numberOfBlocksSent = updateBridgeBtcBlockchain();
-            logger.debug("[updateBridge] Updated bridge blockchain with {} blocks", numberOfBlocksSent);
+            logger.trace("shouldUpdateBridgeBtcBlockchain: " + shouldUpdateBridgeBtcBlockchain);
+            if(shouldUpdateBridgeBtcBlockchain) {
```

Comment:
Not a big deal, but maybe the `if` should go outside the try block

Reply:
Updated!